objects_create_space_test.go

```diff
@@ -95,7 +95,7 @@ func TestCreateSpaceResponseValuePass(t *testing.T) {
 	jsonBytes := []byte(`{"status":200,"data":{"id":"id2","name":"name","description":"desc","custom":{"a":"b"},"created":"2019-08-19T15:05:51.493894Z","updated":"2019-08-19T15:05:51.493894Z","eTag":"Aee9zsKNndXlHw"}}`)
 
 	r, _, err := newPNCreateSpaceResponse(jsonBytes, opts, StatusResponse{})
-	assert.Equal(200, r.Status)
+	//assert.Equal(200, r.status)
```

Comment:
Should this be commented out or just removed?

Reply:
Removed